server/registry/actions_deployments.go

```diff
@@ -49,16 +49,17 @@ func (s *RegistryServer) createDeployment(ctx context.Context, name names.Deploy
 		return nil, status.Error(codes.Unavailable, err.Error())
 	}
 
+	if err := name.Validate(); err != nil {
+		return nil, status.Error(codes.InvalidArgument, err.Error())
+	}
+
+	// SDG: can we get the create to do this check?
```

Comment:
nit: I don't think we have any other personal identifiers (initials) in the code - "git blame" makes attribution easy enough :) Also, this comment is in essentially the same place as the one that's added to line 60 of actions_specs.go. Could these be identical? Just trying to reduce the burden on future readers of these two files, which should have a high level of similarity. Maybe someday someone will merge them with generics :)

Reply:
Oops. That was just an oversight - I had meant to review that, not push it. (I had stopped using TODO because there were too many to easily track in the codebase.)